src/Tools/CoreFx.Tools/GenerateBindingRedirect.cs

```diff
@@ -94,14 +94,16 @@ public override bool Execute()
                 {
                     XElement codeBase = new XElement(ns + "codeBase",
                                                 new XAttribute("version", result.Version),
-                                                new XAttribute("href", Path.GetFullPath(assembly.ItemSpec)));
+                                                new XAttribute("href", Path.GetFileName(assembly.ItemSpec)));
```

Comment:
Actually yes we can eliminate all of the <assemblyBinding> element and the config would only be like this:

```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
  <runtime>
    <developmentMode developerInstallation="true" />
  </runtime>
</configuration>
```

That is enough as it will use DEVPATH to locate all its dependencies. Will change that